frontend/src/components/details/navbar.css

```diff
@@ -4,7 +4,7 @@
     height: auto;
     color: #1b1534;
     padding: 10px;
-    background-color: var(--global--button--color) !important;
+    background-color: #ffffff !important;
```

Comment:
I'd prefer having a variable for the color and the background-color. Is that possible?

Reply:
We can change that😄